### ISSUE TITLE
Inconsistant keyboard navigation and missing accessability features

### ISSUE THREAD

**Issue by JouanDeag** [wisp-gg/frontend#436]
### Describe the bug

When using TAB to navigate pages, the behaviour of what gets navigated is inconsistent and sometimes missing.

For example, the "Manage" feature on the index page has both the button and tooltip tabbable while the "Console" feature only has the button. It is also not possible to reach and copy the allocation of a server using keyboard navigation.

Once managing a server, most of the buttons are not reachable via only a keyboard. There's a "ghost" tab between all expanded buttons in the sidebar and the drawers are not able to be reached in order to be expanded / collapsed which makes use impossible.

### Expected Behavior

All features should be accessible to keyboard-only users and the areas lacking, as described above, should be improved.

### Actual Behavior

See above.

### Steps to Reproduce

- Visit the panel
- Use keyboard navigation (TAB)

### Browser Details

Chrome 120

### Operating System

Windows 10
